rs/sns/governance/canister/governance_test.did

```diff
@@ -820,7 +820,10 @@ type UpgradeJournal = record {
   entries : vec UpgradeJournalEntry;
 };
 
-type GetUpgradeJournalRequest = record {};
+type GetUpgradeJournalRequest = record {
+  max_entries : opt nat64;
+  start_index : opt nat64;
```

Comment:
It seems like SNS-W uses:
```
  limit : nat32;
  starting_at : opt SnsVersion;
```

Which seems fine. I like `starting_at` more than `offset` because it I think it's a bit more clear that it's an index (from the beginning), unlike offset which to me is ambiguous whether it's offset from the beginning or the end.